.github/workflows/main.yml

```
@@ -12,6 +12,7 @@ jobs:
   test:
     name: ${{ matrix.task }} (${{ matrix.os }}, ${{ matrix.compiler }}, R ${{ matrix.r_version }}, ${{ matrix.build_type }})
     runs-on: ${{ matrix.os }}
+    timeout-minutes: 60
```

I don't think we need it in the `master`.
```suggestion
    runs-on: ${{ matrix.os }}
```

---

why not? What is the problem with including it?

The default is 6 hours, which is way longer than these jobs need

---

There are dozens of config params. Including them all will make our file less readable. I believe we should keep configs as much clean and short as it is possible. Normally, our jobs don't need neither 1 hour nor 6 hours. So I don't see any differences why we should change this param from 360 to 60. Pending status more than 20 mins already indicates that something is wrong with our builds (for the current number of tests).

---

I respectully disagree but it is not worth holding up this blocking PR. GitHub won't allow me to commit this suggestion in the browser, but I'll commit it in a few hours when I am home.